### DISCUSSION TITLE
Manual/CacheServer

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#385] [Notes]
# Manual/CacheServer

https://docs.unity3d.com/Manual/CacheServer.html

**Comment by rthery** [comment]
Please note that you should not use different unity versions at the same time on the same cache server. You will end up corrupting your assets (very problematic if you use asset bundles in production)  
If all your unity version are 2018.1+, you can create a different cache server instance per unity version, each accessible with a specific port.
